packages/astro/src/core/config.ts

```diff
@@ -81,6 +138,37 @@ export const AstroConfigSchema = z.object({
 /** Turn raw config values into normalized values */
 export async function validateConfig(userConfig: any, root: string): Promise<AstroConfig> {
 	const fileProtocolRoot = pathToFileURL(root + path.sep);
+	// Manual deprecation checks
+	/* eslint-disable no-console */
+	if (userConfig.renderers) {
```

Comment:
Worth calling out if `userConfig.renderers` is `undefined` as well?

Reply:
do you mean to detect the value `undefined` vs. not being defined on the object at all? I don't think this is an issue that we need to handle but I'll update with `userConfig.hasOwnProperty('renderers')` just in case.